src/App.js

```diff
@@ -1,15 +1,25 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.